### ISSUE TITLE
weird mouse accelleration

### ISSUE THREAD

**Issue by ghost** [hrvach/deskhop#116]
Awesome project!

Just got my box delivered today and love it! Byebye Input-Leap (which is a great project too). I have an issue with the mouse accelleration though. Is there a toggle to turn it off? I need a flat mouse profile. Also Y needs way more motion than X to travel the same distance for me. Logitech MX518 Mouse.

(I don't code C. I just copied the prebuilt images to the Picos.)

**Comment by hrvach** [comment]
Yes, you can configure all of that in either user_config.h and rebuild or try pre-release version with gui configuration... I yet have to write better documentation for that one but check issues and release notes for more info
